planner/core/point_get_plan_test.go

```diff
@@ -429,3 +429,22 @@ func (s *testPointGetSuite) TestBatchPointGetPartition(c *C) {
 	tk.MustExec("delete from t where (a,b) in ((1,1),(2,2),(3,3),(4,4))")
 	tk.MustQuery("select * from t where (a, b) in ((1, 1), (2, 2), (3, 3), (4, 4))").Check(testkit.Rows())
 }
+
+func (s *testPointGetSuite) TestIssue19141(c *C) {
```

Comment:
Should this case be ported back to master branch?

Reply:
Sure!